lib/player.rb

```diff
@@ -0,0 +1,41 @@
+class Player
+  attr_reader :token
+  def initialize(token)
+  @token = token
+  end
+end
+
+module Players
+  class Human < Player
+    def move(board)
+      input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```